Sources/PrimerSDK/Classes/Data Models/CountryCode.swift

```diff
@@ -525,24 +275,49 @@ internal extension CountryCode {
 }
 
 extension CountryCode {
-
-    // todo: enable locale for local languages too.
-    // reference: https://developers.klarna.com/documentation/klarna-payments/in-depth-knowledge/puchase-countries-currencies-locales/
-    var klarnaLocaleCode: String {
-        switch self {
-        case .at, .dk, .fi, .de, .nl, .no, .se, .ch, .us, .au, .gb:
-            return "en-\(self.rawValue)"
-        case .be:
-            return "fr-BE"
-        case .es:
-            return "es-ES"
-        case .it:
-            return "it-IT"
-        default:
-            return "n/a"
+    
+    private static var languageCode: String {
+        Locale.current.languageCode ?? "en"
```

Comment:
Where is this used? You might want to have the language code that the merchant has set first.

Reply:
Yeah there is, via the `PrimerSettings`. But it doesn't apply on this case.